### DISCUSSION TITLE
is there any way i can dump the query data at the infolist function?

### DISCUSSION THREAD

**Discussion by phamduc1823** [filamentphp/filament#7972] [General]
is there any way guys, i really need to do this to see the result, here is my code:

public static function infolist(Infolist $infolist, Table $table): Infolist
    {
        $stmt = DB::table('quiz_contest_results')
            ->select(DB::raw('MAX(point)'))
            ->groupBy('quiz_contest_id');

        $query = DB::table('quiz_contest_results')
            ->select('user_id', 'quiz_contest_id', 'point')
            ->where('id', 1)
            ->whereIn('point', $stmt);

        $result = $query->get();

        return $infolist
            ->schema([
                Components\Section::make('Personal information')
                    ->schema([
                        Components\Split::make([
                            Components\Grid::make(2)
                                ->schema([
                                    Components\Group::make([
                                        Components\TextEntry::make('name'),
                                        Components\TextEntry::make('email'),
                                    ]),

                                    Components\Group::make([
                                        Components\TextEntry::make('email_verified_at')
                                            ->label('Email verification')
                                            ->badge()
                                            ->getStateUsing(fn(User $record
                                            ): string => $record->email_verified_at?->isPast() ? 'Verified' : 'Unverified')
                                            ->color('success'),

                                        Components\TextEntry::make('type_user')
                                            ->label('Type user')
                                            ->badge()
                                            ->colors([
                                                'success'
                                            ]),
                                    ]),
                                ]),
                        ])->from('lg'),
                    ]),

                //Query data is dumped here
            ]);
    }

**Comment by pboivin** [comment]
This doesn't really make sense. Can you describe what you're trying to do in your app? Why do you need to access this query and what are you trying to do with it?

**Reply by phamduc1823** [reply]
Thank you for your attention, I have handled this problem in a different way
